doc/ca65.sgml

```diff
@@ -858,6 +858,55 @@ symbol file via the linker. Unnamed labels are not written to the object file,
 because they don't have a name which would allow to access them.
 
 
+<sect1>String Constants<label id="String_Constants"><p>
+
+String constants must be enclosed within a pair of quotation marks.
+
+Example:
+
+<tscreen><verb>
+      	.byte "SOS V2.3"
+</verb></tscreen>
+
+Only <tt/.DEFINE/ style macros can be used to define a string to be used elsewhere.
+Note that ca65 has support for raw string constants that can span more than one line 
+and can use user defined delimiters. The start of a raw string is defined 
+by the characters <tt>R"(</tt> in place of a double quote (<tt>"</tt>). 
+In this case, only the character pair <tt>)"</tt> will signal the end of the string. 
+Newlines become part of the string.
+
+A custom delimiter may be added as well. The delimiter string follows the same rules 
+as an identifier, but leading characters can be numeric. The custom delimiter should be
+defined between the quote and the opening parenthesis, and again after the closing 
+parenthesis, but before the closing quote.
+
+Example:
+
+<tscreen><verb>
+      	.DEFINE ImportantNotice R"STRING_END(
+      	      	I can put anything here!
+      	      	It will all become part of the string, until ca65 finds the 
+      	      	keyword "STRING_END". But it has to find a ")" first, and a 
+      	      	single '"' at the very end!      	
+      	)STRING_END"
+
+      	.out ImportantNotice
+</verb></tscreen>
+
+Note that there are some pitfalls to be aware of when using this feature:
+
+If the initial or final delimiters are on separate lines, the extra newlines will 
+become part of the string. Everything inside the parenthesis is captured. If the 
+newlines are not added, this can make the string harder to read, and it may not 
+be possible to match the indentation of the first line with the rest of a multiline 
+string. 
+As well, any leading whitespace to match the formatting of the surrounding 
+code will be preserved in the string.
+
+This command is new and must be enabled with the <tt/.FEATURE raw_strings/ command.
+
+  See: <tt><ref id=".FEATURE" name=".FEATURE"></tt>
+
```

Comment:
The patch lines that are above here describe string literals and raw string literals; they don't describe (named) string constants.  Those lines should be moved to the "`Input format`" section of the document.

The sentence about `.DEFINE` should be removed because subsection "`Using macros to define labels and constants`" explains that `.DEFINE` is the only way to create string constants.  (In that subsection, you should expand the name "string constants" to "string constants and raw-string constants".)